lib/asciidoctor-callout-list-block.js

```diff
@@ -0,0 +1,18 @@
+module.exports = function (registry) {
+
+    registry.treeProcessor(function () {
+
+        const self = this
+
+        self.process(function (document) {
+
+            document.findBy({'context': 'olist', 'style': 'calloutlist'}, function (list) {
```

Comment:
Is `olist` meant to be `colist`?

Reply:
Good question.

What I'm doing is taking an ordered list, and converting into a `colist`, so this line is doing a search for any ordered list in the document which has a `calloutlist` style associated with it.

If it finds one then it converts the ordered list into a `colist` which the Asciidoctor parser will see and then add a load of extra tables and styles to make it into a proper `colist`.

So the `olist` is correct because that's what I'm searching for  :-)